dapps/components.vue/src/components/forms/forms.vue

```diff
@@ -86,7 +86,7 @@
           id="evan-form-test-4"
           label="Field 4 - Select"
           type="number"
-          v-model="field4"
+          v-model="select"
```

Comment:
The select parameter isn't defined within the `forms.ts`. Is it correct? ;)

Reply:
Right! I fixed that.